For getting a clean compile and link on Solaris

Not suitable for upstream

--- a/testsuite/gtk/meson.build	2021-06-04 20:22:52.480896445 -0700
+++ b/testsuite/gtk/meson.build	2021-06-04 20:23:30.817449716 -0700
@@ -1,5 +1,3 @@
-gtk_tests_export_dynamic_ldflag = []
-
 if cc.get_id() != 'msvc'
   if os_darwin
     gtk_tests_export_dynamic_ldflag = ['-Wl,-export_dynamic']
@@ -10,6 +8,8 @@
   endif
 endif

+gtk_tests_export_dynamic_ldflag = []
+
 # Available keys for each test:
 #
 #  - 'name': the test name; used for the test and to determine the base

